redis-pq-demo/README.md

```diff
@@ -0,0 +1,17 @@
+# Example run-time
+
+This is a library that builds and deploys an instance of acapy that is configured to act as a mediator, and use REDIS to as a resilient method of managing acapy's event queue. 
+
+How to see a full fledged demo, running a local ledger, and a set of agents using the mediator (with redis)  
```

Comment:
Remember when we were new to this... this line in the documentation would raise a lot of questions and searches - how do i run a local ledger, where do i go to get it?  At a minimum you can put in the super basic setup, git clone... cd to blah, run commmand x, see ledger running at y... And point them at a relevant readme for more information.

As it is, the current `playground` demo points at bcovrin test, so no ledger needed, but assume that will change.

Reply:
I'll make fewer assumptions and provide more details. good point.